concent_api/core/payments/bankster.py

```diff
@@ -263,7 +263,7 @@ def finalize_payment(deposit_claim: DepositClaim) -> Optional[str]:
             assert False
 
         # Bankster puts transaction ID in DepositClaim.tx_hash.
-        deposit_claim.tx_hash = encode_hex(ethereum_transaction.hash)
+        deposit_claim.tx_hash = ethereum_transaction_hash
```

Comment:
Is it no need to use `encode_hex` any more?

Reply:
:+1: done